src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -4808,6 +4808,105 @@ public sealed class EnablePSRemotingCommand : PSCmdlet
     Add-Content -Path $pluginConfigFile -Value ""CORECLRDIR=$PSHOME""
 }}
 
+function Copy-PluginToEndpoint
+{{
```

Comment:
why two curly braces

Reply:
I followed the pattern in the file but single brackets in powershell code given as a string format throw exceptions.

Reply:
These are needed to escape the '{' character, because the string is being used with .net formatting, i.e., string.Format, via PowerShell's -f keyword.  String.Format uses the '{' character to indicate where formatting arguments are placed.  The code has a comment explaining this.